packages/app/components/jsonschemaform/widgets/CKEditorWidget.tsx

```diff
@@ -77,6 +77,7 @@ function CKEditorWidget(props: WidgetProps) {
                 event.editor.setReadOnly(props.readonly || false)
             }}
             onBeforeLoad={CKEDITOR => {
+                CKEDITOR.config.versionCheck = false
```

Comment:
This disables CKEditor from making an HTTP version check—the first step in displaying the "out of date" notification. Locking to a soon-to-be-deprecated version isn't the safest decision, but I don't really thing that we're going to upgrade to a paid product.